tests/test_provenance.py

```diff
@@ -0,0 +1,31 @@
+from unittest import TestCase
+
+from freezegun import freeze_time
+
+from harmony_service_lib.provenance import get_updated_history_metadata
+
+
+@freeze_time('2001-02-03T04:05:06')
```

Comment:
Is the scope of freeze_time limited to just the `TestProvenance` tests? I just want to make sure we don't have make a call to undo it to prevent other tests from being affected.

Reply:
Yeah - this decorator is only being applied to this class, so it should only affect the tests within it.